digital_land/phase/convert.py

```diff
@@ -4,16 +4,28 @@
 import json_stream
 import os
 import os.path
+import re
 import sqlite3
 import subprocess
 import tempfile
 import zipfile
+from packaging.version import Version
 from io import StringIO
 import pandas as pd
 from .load import Stream
 from .phase import Phase
 
 
+def get_gdal_version():
```

Comment:
@eveleighoj Is there a suitable place to re-factor both copies of this function to?